build.gradle

```diff
@@ -182,30 +182,30 @@ task starthere(type: Exec, dependsOn: [copyLocal, copyExtraLibs, copyTestLibs, c
 }
 
 task fastbuild(dependsOn: [
-    ':dbfit-java:core:check',
-    ':dbfit-java:db2:check',
-    ':dbfit-java:derby:check',
-    ':dbfit-java:hsqldb:check',
-    ':dbfit-java:mysql:check',
-    ':dbfit-java:netezza:check',
-    ':dbfit-java:oracle:check',
-    ':dbfit-java:postgres:check',
-    ':dbfit-java:sqlserver:check',
-    ':dbfit-java:teradata:check',
-    ':dbfit-java:derby:integrationTest',
-    ':dbfit-java:hsqldb:integrationTest'
+      ':dbfit-java:core:check'
```

Comment:
Ok for the `fastbuild`.

The pain is on the `integrationBuild` - when I uncomment `':dbfit-java:teradata:integrationTest'` I have to remember to remove the comma from the end of `':dbfit-java:sqlserver:integrationTest'` - always forget!

Ok to have different formatting/style?